thingy53: Enable MCUboot bootloader (#9507)
nrfconnect/sdk-nrf: samples/nrf5340/empty_net_core/README.rst

.. _nrf5340_empty_net_core:

nRF5340: Empty firmware for network core
########################################

.. contents::
   :local:
   :depth: 2

> You can use this sample to generate empty network core firmware.

Reviewer: ```suggestion
The sample demonstrates how to generate an empty network core firmware.
```

Requirements
************

> The sample supports the following development kit:

Reviewer: ```suggestion
The sample supports the following development kits:
```

.. table-from-sample-yaml::

Building and running
********************

.. |sample path| replace:: :file:`samples/nrf5340/empty_net_core`

.. include:: /includes/build_and_run.txt

Testing
=======

> The sample is used to prevent build failures when building MCUboot child image with nRF53 multi image update feature without network core child image in :ref:`ug_multi_image`.

Reviewer: Information placement to be discussed with @divipillai

> The sample is used to prevent build failures when building MCUboot child image with nRF53 multi image update feature without network core child image in :ref:`ug_multi_image`.
> See the :ref:`thingy53_build_pgm_targets` section of Thingy:53 user guide for example use case.

Reviewer: ```suggestion
When building an MCUboot child image with the nRF53 multi-image update feature, the empty network core sample is built together with the application to prevent build failures related to missing Partition Manager definitions in the :ref:`ug_multi_image`.
The sample is used only by the applications that do not use the network core.
```
